services/hanke-service/src/integrationTest/resources/fi/hel/haitaton/hanke/hankeGeometriat.json

```diff
@@ -35,6 +35,6 @@
     ]
   },
   "version": 1,
-  "createdAt": "2020-11-09T15:53:23.1234567+02:00[Europe/Helsinki]",
-  "updatedAt": "2020-11-09T16:53:23.1234567+02:00[Europe/Helsinki]"
+  "createdAt": "2020-11-09T15:53:23.123+02:00[Europe/Helsinki]",
```

Comment:
Wasn't the time zones supposed to be UTC to/from front (and usually better to stick to UTC in database, too, just to avoid the hazzle of daylight savings messups; calculation logic in between doesn't care as long as timezones are either implicitly same or known).
Like Hanke-API is now passing back ""createdAt": "2020-11-20T09:55:08.2493325Z"... though, maybe with less sub-second digits.